CONTRIBUTING.md

```diff
@@ -329,6 +329,11 @@ $ yarn watch & # runs in the background
   [conventionalcommits](https://www.conventionalcommits.org).
   * The title must begin with `feat(module): title`, `fix(module): title`, `refactor(module): title` or
     `chore(module): title`.
+    * `feat`: indicates a feature added (requires tests and README updates)
```

Comment:
Can we also mention that these requirements are not hard requirements and CDK maintainers have the ability to override via labels if they see fit. 

I agree that people are using chore when they should be using fear to skirt these requirements, it would help to know that there are overrides in place.